build-system/babel-plugins/babel-plugin-transform-amp-extension-call/test/fixtures/transform-call/transform-simple-body/input.js

```diff
@@ -14,10 +14,14 @@
  * limitations under the License.
  */
 
-AMP.extension('amp-mustache', '0.2', function(AMP) {
+AMP.extension('amp-mustache', '0.2', function (AMP) {
   AMP.registerTemplate(TAG, AmpMustache);
 });
 
-AMP.extension('amp-gist', '0.1', AMP => {
+AMP.extension('amp-gist', '0.1', (AMP) => {
   AMP.registerElement('amp-gist', AmpGist);
 });
+
+AMP.extension('amp-foo', '0.1', (AMP) =>
+  AMP.registerBentoElement('amp-foo', AmpFoo, class {})
+);
```

Comment:
@jridgewell Do we plan to define `registerBentoElement` at any point? It does not seem like we do.

Reply:
Sorry, this is no longer necessary. The initial design was to have this as part of the Compile Wrapper, but I figured out a way to handle this without defining a new method. We can just use `AMP.registerElement`.

Reply:
What is the way?

Reply:
@samouri I'm assuming this refers to defining a barebones `registerElement`